configure.ac

```diff
@@ -1192,6 +1192,9 @@ if $CPP $CPPFLAGS conftest.c >conftest.out 2>/dev/null; then
   AC_DEFINE_UNQUOTED([ANDROID_API_LEVEL], [$ANDROID_API_LEVEL],
                      [The Android API level.])
 
+  # For __android_log_write in pylifecycle.c.
+  LIBS="$LIBS -llog"
```

Comment:
Is it right to add this here unconditionally?

Reply:
Yes, because this line is only run when building for Android, where libpython will now have a dependency on liblog unconditionally.

Reply:
FTR: there is a 2-space indent here; it may be hard to see that these lines are guarded by the C pre-processor check in L1185. The conftest program used in that check is found in L1175-1183.